tests/screenshot.spec.js

```diff
@@ -0,0 +1,78 @@
+import { test } from "@playwright/test";
+import { argosScreenshot } from "@argos-ci/playwright";
+import { readFileSync } from "fs";
+import * as cheerio from "cheerio";
+import * as fs from "fs";
+import { join } from "path";
+
+// Extract a list of pathnames, given a fs path to a sitemap.xml file
+// Docusaurus generates a build/sitemap.xml file for you!
+export function extractSitemapPathnames(sitemapPath) {
+  const sitemap = fs.readFileSync(sitemapPath).toString();
+  const $ = cheerio.load(sitemap, { xmlMode: true });
+  const urls = [];
+  $("loc").each(function handleLoc() {
+    urls.push($(this).text());
+  });
+  return urls.map((url) => new URL(url).pathname);
+}
+
+// Converts a pathname to a decent screenshot name
+export function pathnameToArgosName(pathname) {
+  return pathname.replace(/^\/|\/$/g, "") || "index";
+}
+
+// Constants
+const siteUrl = "http://localhost:3000";
+const sitemapPath = "./build/sitemap.xml";
+const stylesheetPath = join(__dirname, "screenshot.css");
+const stylesheet = readFileSync(stylesheetPath).toString();
+
+// Wait for hydration, requires Docusaurus v2.4.3+
+// Docusaurus adds a <html data-has-hydrated="true"> once hydrated
+// See https://github.com/facebook/docusaurus/pull/9256
+function waitForDocusaurusHydration() {
+  return document.documentElement.dataset.hasHydrated === "true";
+}
+
+// Calculate and display progress of the running tests
+function calculateProgress(index, total, pathname) {
+  const percentage = Math.floor((index / total) * 100);
+  const progressBarWidth = 30; // Width of the progress bar in characters
+  const filledSquares = Math.floor((percentage / 100) * progressBarWidth);
+  const emptySquares = progressBarWidth - filledSquares;
+  const progressBar = '█'.repeat(filledSquares) + '░'.repeat(emptySquares);
+
+  console.log(`Progress: [${progressBar}] ${percentage}% (${index + 1}/${total}) - Testing: ${pathname}`);
+}

```

Comment:
improved view so the CI now looks like this 
<img width="967" alt="image" src="https://github.com/user-attachments/assets/72c6d4e1-0b4a-473c-a3e8-0f54d2a984c7" />